notebook/oai_litellm.ipynb

```diff
@@ -0,0 +1,470 @@
+{
```

Comment:
This can probably be reworked into an example notebook for the new LiteLLM capability